examples/nginx-ha/main.tf

```diff
@@ -0,0 +1,57 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+module "nginx-ha" {
+  source                 = "equinix-labs/network-edge-device-nginx/equinix"
+  name                   = "terraform-test-NGINX-ha"
+  hostname               = "terraform-nginx-ha"
+  metro_code             = var.metro_code_primary
+  account_number         = "123456"
+  platform               = "small"
+  software_package       = "STD"
+  term_length            = 1
+  notifications          = ["test@test.com"]
+  additional_bandwidth   = 50
+  mgmt_acl_template_uuid = equinix_network_acl_template.nginx-pri.id
+  ssh_key = {
+    userName = "johndoe-primary"
+    keyName  = equinix_network_ssh_key.johndoe.name
+  }
+  secondary = {
+    enabled                = true
+    metro_code             = var.metro_code_secondary
+    hostname               = "vsrx-sec"
+    account_number         = "123765"
+    additional_bandwidth   = 50
+    mgmt_acl_template_uuid = equinix_network_acl_template.nginx-sec.id
+  }
+}
+
+resource "equinix_network_ssh_key" "johndoe" {
+  name       = "johndoe-secondary"
+  public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDpXGdxljAyPp9vH97436U171cX2gRkfPnpL8ebrk7ZBeeIpdjtd8mYpXf6fOI0o91TQXZTYtjABzeRgg6/m9hsMOnTHjzWpFyuj/hiPuiie1WtT4NffSH1ALQFX//zouBLmdNiYFMLfEVPZleergAqsYOHGCiQuR6Qh5j0yc5Wx+LKxiRZyjsSqo+EB8V6xBXi2i5PDJXK+dYG8YU9vdNeQdB84HvTWcGEnLR5w7pgC74pBVwzs3oWLy+3jWS0TKKtflmryeFRufXq87gEkC1MOWX88uQgjyCsemuhPdN++2WS57gu7vcqCMwMDZa7dukRS3JANBtbs7qQhp9Nw2PB4q6tohqUnSDxNjCqcoGeMNg/0kHeZcoVuznsjOrIDt0HgUApflkbtw1DP7Epfc2MJ0anf5GizM8UjMYiXEvv2U/qu8Vb7d5bxAshXM5nh67NSrgst9YzSSodjUCnFQkniz6KLrTkX6c2y2gJ5c9tWhg5SPkAc8OqLrmIwf5jGoHGh6eUJy7AtMcwE3iUpbrLw8EEoZDoDXkzh+RbOtSNKXWV4EAXsIhjQusCOWWQnuAHCy9N4Td0Sntzu/xhCZ8xN0oO67Cqlsk98xSRLXeg21PuuhOYJw0DLF6L68zU2OO0RzqoNq/FjIsltSUJPAIfYKL0yEefeNWOXSrasI1ezw== john@hades"
```

Comment:
In your README.md and examples, avoid using any credentials that look like real credentials. These credentials will be flagged and reviewed by the security team.

In the README files in each example, the defined steps are assuming there is nothing to modify to make it work: clone -> init -> apply, so if there's something that must be modified like that public_key you should better define a variable or use the terraform tls provider to generate a new one in the example. On the other hand, you can reduce the variables in the examples, like `metro_code_primary`, you can take any one like "SV" so users can get an idea of the expected format for that field

Reply:
@schadalawada if you don't use the Terraform resource to generate the public key, we should use a variable here so that this example is usable without Terraform file modification.  We could then use CI to test this example by defining environment variables to override the SSH key (i.e. `TF_VAR_user_ssh_key`).